src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -17,7 +17,8 @@
 import uk.gov.hmcts.reform.sendletter.entity.Letter;
 import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
 import uk.gov.hmcts.reform.sendletter.model.in.LetterRequest;
-import uk.gov.hmcts.reform.sendletter.services.util.DuplexPreparator;
+import uk.gov.hmcts.reform.sendletter.services.pdf.DuplexPreparator;
+import uk.gov.hmcts.reform.sendletter.services.pdf.PdfCreator;
```

Comment:
I think this should not be part of this PR.This is also imported in other files below

Reply:
package changed, it is needed now